src/sdk/namespace_fs.js

```diff
@@ -537,6 +537,23 @@ class NamespaceFS {
         }
     }
 
+    /**
+     * _should_update_issues_report is intended to avoid updating the namespace issues report in case:
+     * 1. The key doesn't exist and the path is not internal -
+     *    internal path is created for specific cases, for example in version.
+     *    Note: it also covers the delete marker case (since it is in a versioned path)
+     * @param {object} params
+     * @param {string} file_path
+     * @param {object} err
+     */
+    _should_update_issues_report(params, file_path, err) {
+        const { key } = params;
+        const md_file_path = this._get_file_md_path({ key });
+        const non_internal_path = file_path === md_file_path;
```

Comment:
please add a note that this function is correct only for read_object_md, if someone wants to use it on read_object_stream it has a gap regarding directory_content objects- when we get ENOENT error on the content path of the object...